packages/api/utils/sqs.ts

```diff
@@ -0,0 +1,6 @@
+import { SQSClient } from "@aws-sdk/client-sqs";
+// Set the AWS Region.
+const REGION = "us-gov-west-1"; // e.g. "us-east-1"
```

Comment:
Let's not do this. The current region of the Lambda function will always be default for any clients. We don't provide a mechanism to configure/override this. So let's just not provide it.